Sources/MessagingInApp/Gist/Managers/MessageManager.swift

```diff
@@ -33,12 +33,12 @@ class MessageManager: EngineWebDelegate {
             siteId: Gist.shared.siteId,
             dataCenter: Gist.shared.dataCenter,
```

Comment:
shouldn't these 2 also fetched from state?

Reply:
Yup, correct. Updated in #802